README.md

```diff
@@ -1 +1,29 @@
-# minterop-data
+# Minterop Data
+
+## Postgres and Hasura GraphQL/REST API configuration for Mintbase Indexers
+
+Configurations here manage both **postgres database column migrations** and **hasura configuration** in a single script that can be run in one of three environments **sandbox (mainnet NEAR data)**, **testnet (testnet NEAR data)**, and **mainnet**.
+
+Running `scripts/redeploy.sh [env]` can be done locally but will require a {environment_name}.env file will need to be present. Ask if you need help with the contents of this file.
+
+**Note, local migrations should only ever be done run sandbox because they lack the inconsitency checks in CICD**
+
+## Migrations
+
+Migrations under the hood use [diesel](https://diesel.rs/), a Rust library for database manipulation. You will need to install this via cargo to test migrations locally.
+
+<br>
```

Comment:
huh? :D

Reply:
Still no good way in markdown to add line breaks without using HTML